components/push/android/src/test/java/mozilla/appservices/push/PushTest.kt

```diff
@@ -141,12 +141,12 @@ class PushTest {
 
```

Comment:
The changes in this file are the extent of the breaking changes...

- `A-C` will have to turn the `List<Byte>` into a `ByteArray` (what `decrypt` used to return) if they'd like using `toByteArray`. It was either this, or have a small wrapper to do the conversion, I opted for the "less android code in a-s" approach
- `channelID` -> `channelId`: this is trivial but interesting, I tried setting it to `channel_ID`, but the rust compiler gave me warnings to turn it into `channel_id` to make it snake case. So either we make the breaking change, or we can silence the rust compiler... I almost opted for the latter, but wanted to get a second opinion